client/flutter/lib/components/home_page_sections/home_page_donate.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/cupertino.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
+import 'package:who_app/generated/l10n.dart';
+
+class HomePageDonate extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: <Widget>[
+        Positioned.fill(
+          child: _DonateBackground(),
+        ),
+        IntrinsicHeight(
+          child: Padding(
+            padding: EdgeInsets.fromLTRB(0.0, 72.0, 0.0, 60.0),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: <Widget>[
+                Padding(
+                  padding: const EdgeInsets.symmetric(
+                      horizontal: 72.0, vertical: 20.0),
+                  child: ThemedText(
+                    // TODO: localize
+                    'Help support the relief effort',
+                    variant: TypographyVariant.h2,
+                    style: TextStyle(
+                      color: CupertinoColors.white,
+                    ),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                CupertinoButton(
```

Comment:
PageButton for material/inkwell?

Reply:
The design of this button is pretty different from the PageButton (see screenshot below) and doesn't have a background. @davidkaneda would you like this button to have an ink splash?

<img width="376" alt="Screen Shot 2020-05-01 at 4 42 50 PM" src="https://user-images.githubusercontent.com/1484366/80849155-cef46800-8bca-11ea-9648-9090fe727a82.png">

Reply:
Ah ok, its more  like a link than a button.  Nvm then!